mobile/src/main/java/org/hawkular/client/android/HawkularApplication.java

```diff
@@ -22,16 +22,21 @@
 import android.app.Application;
 import android.os.StrictMode;
 
+import com.squareup.leakcanary.LeakCanary;
+
 import timber.log.Timber;
 
 public final class HawkularApplication extends Application {
     @Override
     public void onCreate() {
         super.onCreate();
 
+        if (LeakCanary.isInAnalyzerProcess(this)) {
```

Comment:
@garg-anuj 
Sorry for not replying, I missed this comment.
I will update you on this tonight.